awx/main/tests/docs/test_swagger_generation.py

```diff
@@ -100,12 +100,12 @@ def test_sanity(self, release, request):
         # for a reasonable number here; if this test starts failing, raise/lower the bounds
         paths = JSON['paths']
         assert 250 < len(paths) < 375
-        assert set(list(paths['/api/'].keys())) == set(['get', 'parameters'])
-        assert set(list(paths['/api/v2/'].keys())) == set(['get', 'parameters'])
-        assert set(list(sorted(paths['/api/v2/credentials/'].keys()))) == set(['get', 'post', 'parameters'])
-        assert set(list(sorted(paths['/api/v2/credentials/{id}/'].keys()))) == set(['delete', 'get', 'patch', 'put', 'parameters'])
-        assert set(list(paths['/api/v2/settings/'].keys())) == set(['get', 'parameters'])
-        assert set(list(paths['/api/v2/settings/{category_slug}/'].keys())) == set(['get', 'put', 'patch', 'delete', 'parameters'])
+        assert set(list(paths['/api/{var}'].keys())) == set(['get', 'parameters'])
```

Comment:
What is this doing?  `{var}` looks like string templating, but I see no f-string or `.format()` in the changeset here.